molo/core/tests/test_cas.py

```diff
@@ -84,7 +84,7 @@ def test_succesful_login(self, mock_verify):
         mock_verify.return_value = (
             'test@example.com',
             {'ticket': 'fake-ticket', 'service': service, 'has_perm': 'True',
-             'is_admin': 'True'},
+             'is_admin': 'True', 'email': 'test@example.com'},
```

Comment:
So on `line 94`, can you get the user `User.objects.get(username='test@example.com')` and assert that the returned user has the email address on the email field.